packages/nimble-components/src/select/index.ts

```diff
@@ -837,6 +944,23 @@ export class Select
         this.committedSelectedOption = options[this.selectedIndex];
     }
 
+    private toggleNewActiveOption(getNewActiveIndex: () => number): void {
```

Comment:
Is there a reason why `toggleNewActiveOption` takes a function rather than simply a number? It isn't clear to me why all the callers are creating functions to wrap the logic for determining the new active index.

Reply:
I think I was just over-complicating things in my head. Simplified now.